libpromises/bootstrap.c

```diff
@@ -353,3 +356,31 @@ bool MasterfileExists(const char *masterdir)
 
     return true;
 }
+
+static char *BootstrapIDFilename(const char *workdir)
+{
+   return StringFormat("%s%cbootstrap_id.dat", workdir, FILE_SEPARATOR);
+}
+
+bool CreateBootstrapIDFile(const char *workdir)
+{
+    char *filename = BootstrapIDFilename(workdir);
+
+    FILE *file = fopen(filename, "w");
+    if (file == NULL)
+    {
+        Log(LOG_LEVEL_ERR, "Unable to write bootstrap id file '%s' (fopen: %s)", filename, GetErrorStr());
+        free(filename);
+        return false;
+    }
+    CryptoInitialize();
+    unsigned char buf[8];
+    RAND_bytes(buf, 8);
```

Comment:
Any reason for not using at least 128 bits? I know `2**64` is a big number, but some people are paranoid about these things.

Reply:
Increased to 240 bits, which is divisible by 6, meaning there won't be base64 padding.